infinity/custom_items/general/shraken-Harnes-custom_saare.json

```
@@ -1,6 +1,6 @@
 {
-  "ckey" :            "jacking",
-  "character_name" :  "Mammoth Nestor",
+  "ckey" :            "shraken",
+  "character_name" :  "Ethan Harnes",
   "item_path" :       "/obj/item/clothingbag/custom_saare",
   "req_access" :      [ "ACCESS_SECURITY" ]
   }
```

Будет очень классно, если в конце файла ты добавишь ещё пустую строчку, чтобы пропал красный значок после фигурной скобки
```suggestion
  }

```